modules/installation-vsphere-machines.adoc

```diff
@@ -157,26 +165,18 @@ $ export IPCFG="ip=<ip>::<gateway>:<netmask>:<hostname>:<iface>:none nameserver=
 ----
 $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameserver=8.8.8.8"
 ----
-
-... Set the `guestinfo.afterburn.initrd.network-kargs` property before booting a VM from an OVA in vSphere:
-+
-[source,terminal]
-----
-$ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
-----
 +
-*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*. Ensure that your VM's CPU and memory reservation have the following values:
-**** Memory reservation value must be equal to its configured memory size.
-**** CPU reservation value must be at least the number of low latency virtual CPUs multiplied by the measured physical CPU speed.
-*** Click *Edit Configuration*, and on the *Configuration Parameters* window, search the list of available parameters for steal clock accounting (`stealclock.enable`). If it is available, set its value to `TRUE`. Enabling steal clock accounting can help with troubleshooting cluster issues.
+*** Click *Edit Configuration*, and on the *Configuration Parameters* window, search the list of available parameters for steal clock accounting (`stealclock.enable`). Set the parameter to the value of `TRUE`. Enabling steal clock accounting can help with troubleshooting cluster issues.
 *** Click *Add Configuration Params*. Define the following parameter names and values:
-**** `guestinfo.ignition.config.data`: Locate the base-64 encoded files that you created previously in this procedure, and paste the contents of the base64-encoded Ignition config file for this machine type.
-**** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
 **** `disk.EnableUUID`: Specify `TRUE`.
 **** `stealclock.enable`: If this parameter was not defined, add it and specify `TRUE`.
+**** Create a child resource pool from the cluter's root resource pool. Locate the base-64 encoded files that you created previously in this procedure, and paste the contents of the base64-encoded Ignition config file for this machine type.
```

Comment:
ignition has no association with resource pools

ignition is set per virtual machine regardless of location in vsphere